### PR TITLE
Initial release of Phantasmagorika Quests (Walkthrough conversion).

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: None

* **Server Mode**: Renewal

* **Description of Pull Request**:

# Phantasmagorika Quests
- [x] Memories Quests

Todo Instances:
- [x] The Last Room
- [x] Central Laboratory
- [ ] ~~Charleston Crisis~~ Another PR

- [ ] Mobs skills

### CODE REVIEW

**Comment by Lemongrass3110** [comment]
Item IDs 6961 & 6962 are missing, that is why the CI tests fail right now.

**Comment by jenkijo** [comment]
Good work! I'll help you test it

**Comment by jenkijo** [comment]
I think you are missing ',' at Mob Skill ID 3074

```
3074,Time Holder@NPC_SUMMONSLAVE,idle,196,5,1000,700,10000,no,self,slavele,0,2917,,,,,,
```

I fixed and it work very fine
```
3074,Time Holder@NPC_SUMMONSLAVE,idle,196,5,1000,700,10000,no,self,slavele,0,2917,,,,,,,,
```

**Comment by Lemongrass3110** [comment]
@jenkijo has to be an issue on your end. If an error would have been issued by the buildbot the integration tests would have failed.
It's loading correctly on the repository's branch as you can see here:
https://travis-ci.org/rathena/rathena/jobs/256146329#L1046

**Comment by jenkijo** [comment]
Oh, sorry you. This is my mistake

**Comment by rofasindo** [comment]
I think this wrong mobid, verus daily quest "Eliminating Risk" after kill monster not counting. http://irowiki.org/wiki/Verus_Daily_Quests#Eliminating_Risks

**Comment by rofasindo** [comment]
I have error when I load Last Room instance, missing function close3
lastroom

File: npc/re/instances/LastRoom.txt [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+		cutin "",255;
+	}
+	end;
+
+OnInstanceInit:
+	'map_name$ = instance_mapname("1@uns");
+	'protocole = 0;
+	disablenpc instance_npcname("protocole1");
+	disablenpc instance_npcname("Tamarin#room2");
+	disablenpc instance_npcname("zone_1_2_last_room");
```
Comment (Valorro) [inline npc/re/instances/LastRoom.txt]: @Atemo boss, i cant find this npc "zone_1_2_last_room", do i miss something?

Comment (Questune09) [reply npc/re/instances/LastRoom.txt]: i think thats a map name maybe need a new map for that

File: npc/re/custom/phantasmagorica/phantasm_quests.txt [hunk 2]
```diff
@@ -0,0 +1,9035 @@
...
+				close;
+			case 2:
+				mes "Things seems to have settled.";
+				mes "The workers will resume their work soon.";
+				mes "They're not fighters, so they're usually passive in times like this.";
+				next;
+				mes "[Louis]";
+				mes "Thank you for your hard work.";
+				mes "Our workers will be grateful to you for keeping them safe.";
+				erasequest 7613,7614;// Eliminating Risks -> Eliminating Risks - Down
```
Comment (anity99) [inline npc/re/custom/phantasmagorica/phantasm_quests.txt]: `changequest 7613,7614;`

**Comment by ecdarreola** [comment]
Which entry for instance_db should be followed after also pulling Devil Tower? Currently pulling all this for testing but there are conflicting files in instance.

**Comment by Atemo** [comment]
This merge should resolve most of the conflict @ecdarreola

**Comment by ecdarreola** [comment]
@Atemo : can you update its emoticon list? The emotion script causes error. Thanks!

File: npc/re/custom/phantasmagorica/phantasm_quests.txt [hunk 3]
```diff
@@ -0,0 +1,9068 @@
...
+			case 13:
+				npctalk "Hello. This is Silvia Loren.";
+				sleep2 4000;
+				npctalk "My hobby is chicken. I like beer~.";
+				sleep2 4000;
+				npctalk "Eh... Kind of went overboard by taking on a project today.";
+				sleep2 4000;
+				npctalk "Honestly it isn't a subject I was interested in.";
+				sleep2 4000;
+				npctalk "Vi
```
Comment (Jeybla) [inline npc/re/custom/phantasmagorica/phantasm_quests.txt]: Unknown characters

File: npc/re/custom/phantasmagorica/memory_record.txt [hunk 4]
```diff
@@ -0,0 +1,2579 @@
...
+			mes "Since I can't figure out the infection route...";
+			next;
+			mes "[Nurse Lapplad]";
+			mes "We have to prepare for a possible disaster...";
+			next;
+			mes "[Nurse Lapplad]";
+			mes "Anyway, Adventurer~ Assuring and the best!";
+			next;
+			mes "[Nurse Lapplad]";
+			me
```
Comment (Jeybla) [inline npc/re/custom/phantasmagorica/memory_record.txt]: Unknown characters

**Comment by Jeybla** [comment]
Merged master and updated the emotion calls. @Atemo above I mentioned two lines of unknown characters, I just removed in my last commit. Don't forget to pull the changes.

**Comment by ecdarreola** [comment]
Tested the PR and no problem from the script. Also let the players tested the instance for checking. They confirm working as intended.

**Comment by Atemo** [comment]
Thanks for the feedback @ecdarreola :+1:

**Pull request merged**